nncf/quantization/quantize.py

```diff
@@ -78,3 +82,55 @@ def quantize(model: TModel,
                              fast_bias_correction, model_type, ignored_scope)
 
     raise RuntimeError(f'Unsupported type of backend: {backend}')
+
+
+def quantize_with_accuracy_control(model: ModelType,
+                                   calibration_dataset: Dataset,
+                                   validation_dataset: Dataset,
+                                   validation_fn: Callable[[Any, Iterable[Any]], float],
+                                   max_drop: float = 0.01,
+                                   drop_type: DropType = DropType.ABSOLUTE,
+                                   preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
+                                   target_device: TargetDevice = TargetDevice.ANY,
+                                   subset_size: int = 300,
+                                   fast_error_correction: bool = True,
+                                   model_type: Optional[str] = None,
+                                   ignored_scope: Optional[IgnoredScope] = None) -> ModelType:
```

Comment:
Do we have a ticket or plans to add `additional options` for the AA implementation?

Reply:
I created the ticket (Ref: 96509).